README.md

```diff
@@ -1,86 +1,113 @@
+<br><br>
 
-**Boss Room's min spec devices are:**
-- iPhone 6S
-- Samsung Galaxy J2 Core <br>
+---
 
-Find our full project documentation, [here](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom/bossroom/index.html).
+### 💡 Utilities Package
+This repository also contains a [Utilities](Packages/com.unity.multiplayer.samples.coop) package, containing reusable sample scripts. You can install it using the following manifest file entry:
+<br>
+`"com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop",`
 
-To connect and find support, join the multiplayer community on the Unity [Discord](https://discord.gg/mNgM2XRDpb) and [Forum](https://forum.unity.com/forums/multiplayer.26/).<br> Got feedback? Tell us what you think using our new [Feedback Form](#feedback-form) <center>.
-| -- |
+---
 
 
+<br>
 
-## Contents and Quick Links
+For more information on the art of Boss Room, see [ART_NOTES.md](Documentation/ART_NOTES.md).
 
-<!-- TOC generated from https://luciopaiva.com/markdown-toc/ -->
 
+![](Documentation/Images/Boss.png)
+<br><br>
+  
+-----
+
+## Readme Contents and Quick Links
+<!-- TOC generated from https://luciopaiva.com/markdown-toc/ -->
+<details open>
+<summary> <b>Click to expand/collapse contents</b> </summary>
 
-- [Boss Room Overview](#boss-room-overview)
-- [Getting the project](#getting-the-project)
+- ### [Getting the project](#getting-the-project)
   - [Direct download](#direct-download)
   - [Installing Git LFS to clone locally](#installing-git-lfs-to-clone-locally)
-- [Registering the project with Unity Gaming Services (UGS)](#registering-the-project-with-unity-gaming-services-ugs)
-- [Opening the project for the first time](#opening-the-project-for-the-first-time)
-- [Exploring the project](#exploring-the-project)
-- [Testing multiplayer](#testing-multiplayer)
-- [Index of resources in this project](#index-of-resources-in-this-project)
+- ### [Requirements](#requirements)
+  - [Min Spec Devices](#boss-rooms-min-spec-devices-are)
+- ### [Opening the project for the first time](#opening-the-project-for-the-first-time) >
+- ### [Exploring the project](#exploring-the-project)
+  - [Registering with Unity Gaming Services (UGS)](#registering-the-project-with-unity-gaming-services-ugs)
+- ### [Testing multiplayer](#testing-multiplayer) 
+  - [Local Multiplayer Setup](#local-multiplayer-setup)
+  - [Multiplayer over Internet](#multiplayer-over-internet)
+  - [Relay Setup](#relay-setup) 
+- ### [Index of resources in this project](#index-of-resources-in-this-project)
   - [Gameplay](#gameplay)
   - [Game Flow](#game-flow)
   - [Connectivity](#connectivity)
   - [Services (Lobby, Relay, etc)](#services-lobby-relay-etc)
   - [Tools and Utilities](#tools-and-utilities)
-- [Troubleshooting](#troubleshooting)
-- [Licence](#licence)
-- [Other samples](#other-samples)
+- ### [Troubleshooting](#troubleshooting)
```

Comment:
I was testing the header links and some of them don't direct to the right spot in the readme. This line is one example.